### DISCUSSION TITLE
underestimated SFRs

### DISCUSSION THREAD

**Discussion by getrude386** [mboquien/cigale#77] [Q&A]
Hi, can you please assist with possible reasons for getting underestimated SFRs for the radio-selected galaxies. I have attached a distribution of results I got when using cigale and AGNfitter. But cigale produces underestimated SFRs for other sources i.e. they do not fall within the expected range of Whittam et al. (2022) as AGNfitter results. I have also attached my pcigale.ini file and my input data.

[pcigale (1).txt](https://github.com/mboquien/cigale/files/13871923/pcigale.1.txt)
![Screenshot from 2024-01-09 11-41-58](https://github.com/mboquien/cigale/assets/105718721/f2403e97-8853-4878-ad85-9813ea444cf3)
[100sources_normalized_all_sources.txt](https://github.com/mboquien/cigale/files/13871973/100sources_normalized_all_sources.txt)

**Comment by mboquien** [comment]
The estimate of the SFR would be very dependent on the assumed SFH and whether you used the best-fit or Bayesian estimates. I suspect if you adopt an SFH that does not drop to ~0, the results will be similar.
